### ISSUE TITLE
Conjunctival subepithelial fibrosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9974]
**Preferred term label:**
Conjunctival subepithelial fibrosis

**Synonyms**
Scarring

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C3553304

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mucous membrane pemphigoid

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001090
- label: Conjunctival subepithelial fibrosis
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: PMID:28789979
- def: fine white linear scars under intact conjunctival epithelium. This scarring appears in areas of previous conjunctival ulceration or pseudomembrane formation, weeks to months after the insult is healed.
- comment: Subepithelial fibrosis can be a manifestation of ocular graft-versus-host disease (oGVHD), but can also be found after epidemic keratoconjunctivitis, Stevens-Johnson syndrome, and chemical burns.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9974
- synonyms:
